playground/nitro.config.ts

```diff
@@ -1 +1,14 @@
-export default defineNitroConfig({});
+import { defineEventHandler } from "h3";
+
+export default defineNitroConfig({
```

Comment:
Please consider not pushing commits for playground changes. They are for local testing only